lib/workers/repository/process/vulnerabilities.spec.ts

```diff
@@ -0,0 +1,74 @@
+import type { Ecosystem, OsvOffline } from '@jamiemagee/osv-offline';
```

Comment:
I see we're grabbing stuff from @JamieMagee's repository here. 😄 

Maybe we should make a official `renovate/osv-offline` repository, and link to that instead? That way the Renovate organization has control over what we link to?

Reply:
Yes, absolutely. I think it should either be owned by the `renovate` org or `ossf`[^1] org in the long run.

The OpenSSF question is still open for debate. Currently the OSV schema is owned by the OpenSSF[^2]. However, OSV the service is owned by Google[^3] right now. Discussion around donating OSV to OSSF is currently ongoing in the OpenSSF vulnerability disclosures working group[^4]

In the short-term, owning the project in my repo allows me to rapidly iterate, but we should take a decision for the long term when we look to enable this feature by default.

[^1]: https://github.com/ossf
[^2]: https://github.com/ossf/osv-schema
[^3]: https://github.com/google/osv
[^4]: https://github.com/ossf/wg-vulnerability-disclosures

Reply:
I created #14605 to track the issue.